src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Legacy/TokenizerBackedParser.cs

```diff
@@ -14,10 +14,16 @@ namespace Microsoft.AspNetCore.Razor.Language.Legacy;
 internal abstract class TokenizerBackedParser<TTokenizer> : ParserBase
     where TTokenizer : Tokenizer
 {
+    protected delegate void SpanContextConfigAction(SpanEditHandlerBuilder editHandlerBuilder, ref ISpanChunkGenerator chunkGenerator);
```

Comment:
Nit: do we need a better name than `SpanContext...` given that that isn't a concept anymore? Ditto for the other uses in this file. Or are we fine thinking about it as a conceptual thing.

Reply:
I think we should leave it as is for now. My hope is that these go away entirely at some point.